### DISCUSSION TITLE
Password for Creating Pastes

### DISCUSSION THREAD

**Discussion by serkankster** [PrivateBin/PrivateBin#1085] [Q&A] [answered]
My use-case requires having a password for creating pastes, however reading should not require any password. I wasn't able to find a way of configuring this in PrivateBin. Wondering if someone might be able to let me know if this is possible?

**Comment by r4sas** [comment, marked as answer]
Please read this article: https://github.com/PrivateBin/PrivateBin/wiki/Restrict-upload-using-NGINX

**Comment by serkankster** [comment]
Got it, thank you. No other solution than adding authorization through a reverse proxy, right? I was hoping for a simpler config change  but let me see how i can do this with traefik.
